packages/elements/src/components/MarkdownViewer/CustomComponents/CodeWithSchemaViewer.tsx

```diff
@@ -23,6 +26,48 @@ export const CodeWithSchemaViewer = (props: IComponentMappingProps<ICode<ICodeAn
     return <SchemaAndDescription title={annotations?.title} schema={parsedValue} />;
   }
 
+  if (nodeType === 'http') {
+    if (!isObject(parsedValue) || (!isHttpRequest(parsedValue) && !isHttpOperation(parsedValue))) {
+      return null;
+    }
+
+    return <TryIt httpOperation={isHttpOperation(parsedValue) ? parsedValue : parseHttpRequest(parsedValue)} />;
```

Comment:
Shouldn't we rather just render the `DefaultCode` in the failure path? Not a strong opinion here though.

Reply:
So far we are rendering null for `json_schema`, we did render null in v6 for request maker too, so I didn't change that here. That might be worth considering though, but not sure if should be done here.